scraper/src/Scraper.hs

```diff
@@ -22,7 +23,45 @@ runScrape = do
 scrape :: IO ()
 scrape = do 
   website <- httpLBS "http://leg.colorado.gov/bill-search?field_sessions=10171&sort_bef_combine=field_bill_number%20ASC" 
-  L8.writeFile "test.html" $ getResponseBody website
+  L8.writeFile "test/example/bill_page_1.html" $ getResponseBody website
+
+-- I'm very sorry. Forgive me for my sins.
+fixNextPage = fmap mconcat . fmap mconcat $ fmap fromMaybe' nextPage
+fixLastPage = fmap mconcat . fmap mconcat $ fmap fromMaybe' lastPage
+
+-- This is far from ideal. please change it later 
+fromMaybe' mval = 
+  case mval of 
+    Nothing  -> undefined 
+    Just val -> val
+
+nextPage :: IO (Maybe [[ByteString]]) 
+nextPage = do 
+  website <- readFile "test/example/bill_page_1.html"
+  return $ S.scrapeStringLike website next
+
+    where 
+      next :: S.Scraper ByteString [[ByteString]]
+      next = S.chroots ("li" S.@: [S.hasClass "pager-next"]) link
```

Comment:
`S.chroots` takes a selector and a `Scraper s a` and returns a `Scraper s [a]`, where we search for every element that matches the scraper, and return a list of all the results. I think this is a good function, but I think `link` could be improved, which would change the type signature to `S.Scraper ByteString [Maybe ByteString]`.

Reply:
This is completed.